protobuf/sliverpb/constants.go

```diff
@@ -186,6 +186,8 @@ const (
 	MsgSetEnvReq
 	// MsgSetEnv
 	MsgSetEnv
+	// MsgUnsetEnvReq
+	MsgUnsetEnvReq
```

Comment:
Careful here, this will cause an incompatibility with older implants since it will redefine the values after it.

We can probably just bundle in more fixes and push a v1.5.0 or something.

Reply:
Good catch, I'll move it to the bottom.